src/models/model.ts

```diff
@@ -216,6 +216,23 @@ class Model {
             return resp.json();
         }).then((userKeybindings) => {
             this.keybindManager.setUserKeybindings(userKeybindings);
+            this.testConfigListDir();
+        });
+    }
+
+    testConfigListDir() {
```

Comment:
This is test code and should be removed, but I figured it might help a reviewer to see how I tested this code